liberty-ls/src/main/java/io/openliberty/tools/langserver/diagnostic/LibertyPropertiesDiagnosticService.java

```diff
@@ -76,8 +76,9 @@ private List<Diagnostic> computeInvalidValuesDiagnostic(PropertiesValidationResu
             String lineContentInError) {
         List<Diagnostic> lspDiagnostics = new ArrayList<>();
         if (validationResult.hasErrors()) {
-            String messageTemplate = DiagnosticMessages.getString(validationResult.getInvalidValueMessageTemplate());
-            String message = MessageFormat.format(messageTemplate, validationResult.getValue(), validationResult.getKey());
+            String property = validationResult.getKey();
+            String messageTemplate = DiagnosticMessages.getString(validationResult.getDiagnosticType());
+            String message = MessageFormat.format(messageTemplate, validationResult.getValue(), property, ServerPropertyValues.getIntegerRange(property));
```

Comment:
What if this property does not have a property range? Does it just pass null and it is ignored because the message does not expect a value? Might be worth a comment in the code explaining.

Reply:
Yes, I was looking into reworking this, but currently the `{2}` arg is only used by the Integer messages. I'll add the comment explaining it